plugins/cilium-cni/cilium-cni.go

```diff
@@ -562,11 +564,6 @@ func cmdDel(args *skel.CmdArgs) error {
 		//              Recoverable() will return true if error can be
 		//              retried
 		log.WithError(err).Warning("Errors encountered while deleting endpoint")
-		if clientError, ok := err.(client.ClientError); ok {
-			if clientError.Recoverable() {
```

Comment:
Is something still using `Recoverable()`?

Reply:
Yes, the chained veth plugin is using it.

https://github.com/cilium/cilium/blob/master/plugins/cilium-cni/chaining/generic-veth/generic-veth.go#L215

Should I remove it there as well? Maybe we should avoid returning errors in chained plugins as well.

Reply:
For now I removed it. Let me know if you think that it should remain.